### PR TITLE
update prose to reflect cron time shorthand codes

### PR DESCRIPTION
**Pull request by shawndwells**
Aligns the aide prose with new cron short codes accepted in OVAL per https://github.com/OpenSCAP/scap-security-guide/pull/2723

### CODE REVIEW

File: shared/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking.rule [hunk 1]
```diff
@@ -48,3 +49,5 @@ ocil: |-
     <pre>$ grep aide /etc/crontab</pre>
     The output should return some similiar to the following:
     <pre>05 4 * * * root /usr/sbin/aide --check</pre>
+
+    NOTE: The useage of special cron times, such as @daily or @weekly, is acceptable.
```
Comment (redhatrises) [inline shared/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking.rule]: /useage/usage

Comment (shawndwells) [reply shared/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking.rule]: Thanks - done!

**Comment by yuumasato** [comment]
The oval check also accepts `monthly`, this should be reflected in description.

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by shawndwells** [comment]
Thanks @yuumasato. Updated.

**Comment by redhatrises** [comment]
@shawndwells `@monthly` does not appear in the PR after your update.

**Comment by shawndwells** [comment]
Do we list all possible options, or use language similar to "such as" with a few examples?

**Comment by redhatrises** [comment]
@shawndwells Maybe just go a head and say `such as @hourly, @daily, @weekly, or @monthly`?

**Comment by redhatrises** [comment]
Ack. Re-reading the English is good, and we have a lot of oval that allows for things out of the range by design.

**Comment by yuumasato** [comment]
> and we have a lot of oval that allows for things out of the range by design.

@redhatrises I'm puzzled by this, can you elaborate more?
I would find it really strange if I look at a Rule, and its check is allowing more then described. Specially when rule says, `At a minimum ... At most, ...`

**Pull request merged**